src/corefx/System.Globalization.Native/pal_collation.c

```diff
@@ -365,15 +332,13 @@ static int CanIgnoreAllCollationElements(const UCollator* pColl, const UChar* lp
 
 void CreateSortHandle(SortHandle** ppSortHandle)
 {
-    *ppSortHandle = (SortHandle*)malloc(sizeof(SortHandle));
+    *ppSortHandle = (SortHandle*)calloc(1, sizeof(SortHandle));
```

Comment:
It is non-obvious that calloc is used here to zero-fill. Maybe `malloc` + `memset` would be more self-describing.

Reply:
That's actually a convention I borrowed from CoreFX PAL code, but I am happy to change it since I find malloc+memset more readable here.